File: python/Event.py
import ROOT

class Event:
   """
   Class for tracker as target events.
   """

   def __init__(self, positive_lepton, negative_lepton, pbrem = -1, after_brem = ROOT.TLorentzVector()):
      self.lepplus = lepton1
      self.lepminus = lepton2
      self.plusbrem = pbrem
      self.abrem = after_brem

File: c++/Event.c
#include "TLorentzVector.h"
#include "Event.h"

event::event()
{
}

event::event(TLorentzVector pvec, TLorentzVector mvec, bool posbrem, TLorentzVector pre_vec)
{
   mu_pos_vec = pvec;
   mu_neg_vec = mvec;
   pbrem = posbrem;
   pre_brem_vec = pre_vec;
}

event::event(TLorentzVector pvec, TLorentzVector mvec)
{
   mu_pos_vec = pvec;
   mu_neg_vec = mvec;
}

event::~event()
{
}


